docs/0.4/examples/gameroom/walkthrough/images/auth_login_bubba1.svg

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Missing copyright

Reply:
same for all images

Reply:
Added a copyright to the SVG files.

Reply:
you need to remove the xml line as well, otherwise the images do not show up correctly in the website

Reply:
The other SVG files don't have the xml line or the DOCTYPE line, so I removed both of these lines.